microservice/challenge-1/ReminderAppl/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:8-jdk-alpine as basebuilder
```

Comment:
tried to build the image local and got permission error on `./mvnw`, is it possible to update the multi-stage build using a maven image instead?

![image](https://user-images.githubusercontent.com/13165955/149786775-cd0e7251-b48d-465d-b7ce-27c79968389b.png)

Reply:
you need chmod +x ./mvnw locally.

Reply:
I added RUN chmod +x ./mvnw in the Dockerfile. Also tried with a maven image maven:3.5.2-jdk-8-alpine  and it worked for me locally.